stan/math/rev/core/cmath.hpp

```diff
@@ -0,0 +1,119 @@
+#ifndef STAN_MATH_REV_CORE_CMATH_HPP
+#define STAN_MATH_REV_CORE_CMATH_HPP
+
+#include <stan/math/rev/core/var.hpp>
+#include <stan/math/prim/meta.hpp>
+
+namespace stan {
+namespace math {
+
+/**
+ * Return `true` if the specified argument is negative and `false`
+ * otherwise.
+ *
+ * Overloads `std::signbit` from `<cmath>` for argument-dependent
+ * lookup.
+ *
+ * @tparam T type of argument
+ * @param[in] v argument
+ * @return `true` if the argument is negative
+ */
+template <typename T, require_autodiff_t<T>...>
+inline bool signbit(T&& v) {
+  using std::signbit;
+  return signbit(v.val());
```

Comment:
Yeah since fvar and var have `.val()` we can move this to `prim`. Should we also have each of these for arithmetic types or just rely on the cmath impl for those (in case someone calls `stan::math::signbit(a)` where a is double)?

If we wanted to be v smart we could break these up to have signatures for 

1. Arithmetic types
2. AD types with an arithmetic return from `.val()`
3. AD types with heavier types in `.val()`

Then 1 and 2 would pass by copy while 3 would be able to forward along / pass a reference to `.val()`